xenvm.xml

```diff
@@ -1,16 +1,15 @@
 <manifest>
 
   <!-- Remotes -->
-  <remote  name="github" fetch="https://github.com/xen-troops/"
-           revision="android-12-master" />
+  <remote  name="github" fetch="https://github.com/xen-troops/"/>
```

Comment:
Hi @arminn, update commit message of this commit: https://github.com/xen-troops/android_manifest/pull/86/commits/0594d19869dc4ed4f4c9d6488a5cb59b39650a2d

b>>>a<<<ranch => branch